recipes/csv-parser/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CsvParserConan(ConanFile):
+    name = "csv-parser"
+    description = "Vince's CSV Parser with simple and intuitive syntax"
+    topics = ("conan", "csv-parser", "csv", "rfc 4180", "parser", "generator")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/vincentlaucsb/csv-parser"
+    license = "MIT"
+    settings = "os", "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        tools.check_min_cppstd(self, 11)
```

Comment:
Only check `compiler.cppstd` when it is set. The default is `None`.
```suggestion
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, 11)
```